chsdi/templates/htmlpopup/oev_haltestellen.mako

```diff
@@ -0,0 +1,151 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <%!
+        from pyramid.url import route_url
+        import chsdi.lib.helpers as h
+    %>
+    <%
+        protocol = request.scheme
+        lang = request.lang
+        topic = request.matchdict.get('map')
+        host = h.make_agnostic(request.host_url + request.uscript_name)
+
+        id = c['attributes']['nummer']
+        c['baseUrl'] = h.make_agnostic(''.join((protocol, '://', request.registry.settings['geoadminhost'])))
+    %>
+<style>
+  tr:last-child {
+    display:none;
+  }
+</style>
+<script>
+
+var timesRun = 0;
+var dest;
+function getInfos(val){
+    if (val) {
+      dest = val;
```

Comment:
I was thinking that this would not be an html popup but rather a new component, I don't think we have JS in regular htmlpopups at the moment. Regular htmlpopup would be without JS with a link to the extendedpopup, but in geoadmin we'd handle things diffrently for this layer.
Probably involves some more thinking, but it would be a good start for front-end templating.
(In layersConfig maybe)